core-services/prow/02_config/_config.yaml

```diff
@@ -618,6 +618,20 @@ tide:
             skip-unknown-contexts: true
           release:
             skip-unknown-contexts: true
+      openshift-kni:
```

Comment:
lets also include RDS (https://github.com/openshift-kni/telco-reference) if its not included yet

Reply:
missed that, will add it.